Unix/buildtool

```diff
@@ -514,7 +518,7 @@ if [ "$arg1" = "cc" ]; then
 
     case "$platform" in
 
-        LINUX_IX86_GNU|LINUX_X86_64_GNU)
+        LINUX_*)
```

Comment:
I'm not sure it's a good idea for this to be so "open". Why not just add the platform for Linux on PPC to this list? I think that's safer, FWIW.